retrieval/README.md

```diff
@@ -2,14 +2,15 @@
 The image retrieval pipeline consists of several distinct steps:
 
 ## ResNet50 inference to generate database and query set arrays
-We utilize a native [PyTorch inference](R50_inference.py) to generate the database and query sets for image retrieval. An example for performing 
-retrieval on ImageNet-4K with an MRL pretrained model is provided below:
+We utilize a native [PyTorch inference](../inference/pytorch_eval.py) to generate the database and query sets for image retrieval. An example for performing 
+retrieval on ImageNet-4K with an MRL pretrained model (trained with PyTorch DDP) is provided below:
 
 ```
-python R50_inference.py --model_path=path-to-model/ --dataset_name=imagenet4k --mrl=1 --efficient=0 
+python pytorch_inference.py --retrieval=1 --path=pretrained_model_path/ --retrieval_array_path=output_path/ \
```

Comment:
if someone runs this command from this folder it will not work. get them back to the right folder if you can.